src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/client/CommunityApiClient.kt

```diff
@@ -15,16 +14,15 @@ import java.time.LocalDate
 class CommunityApiClient(@Qualifier("communityWebClientAppScope") private val webClient: WebClient) {
 
   fun getRegistrations(crn: String): Pair<List<Registration>, List<Registration>> =
-    getRegistrationsCall(crn).sortedByDescending { it.startDate }
-      .partition { ComplexityFactor.from(it.type.code) == IOM_NOMINAL }
+    getRegistrationsCall(crn).partition { it.type.code == IOM_NOMINAL.registerCode }
 
-  fun getDeliusAssessments(crn: String): DeliusAssessments? {
+  fun getDeliusAssessments(crn: String): DeliusAssessments {
     return webClient
       .get()
       .uri("/offenders/crn/$crn/assessments")
       .retrieve()
       .bodyToMono(DeliusAssessments::class.java)
-      .block()
+      .block() ?: DeliusAssessments(BigDecimal(0), 0)
```

Comment:
This is what I don't like, but it tidies up everything downstream of it

Reply:
I don't think its the right solution, but if we translated this DTO into a domain object with sensible defaults we'd get the same result.

Reply:
I've been thinking about doing this for a while